### ISSUE TITLE
Forecasts: Prophet Vs ETS forecast: Hyndman

### ISSUE THREAD

**Issue by sunitgk** [facebookincubator/prophet#177]
Forecasts from Prophet versus ETS forecast from Hyndman's forecast package at 50% delta.
Has anyone done a comparison of the forecast accuracy between the 2 packages?

**Comment by sillasgonzaga** [comment]
I suggest you to use my R package called [mafs](https://cran.r-project.org/web/packages/mafs/index.html) to fit 18 models from the forecast package and compare its results with prophet.

**Comment by dashaub** [comment]
Speaking generally, they are targeting slightly different use cases. Prophet will work well for daily data while ets/auto.arima/etc are designed for weekly/monthly/yearly data. Build the right features and the the auto.arima/nnetar models can be applied to any frequency.

The original prophet paper [here](https://facebookincubator.github.io/prophet/static/prophet_paper_20170113.pdf) does a simple comparison that you're wanting, but it applies to 1) a limited type of dataset so won't necessarily generalize to other datasets or timeseries frequencies (see the machine learning [no free lunch theorem](https://en.wikipedia.org/wiki/No_free_lunch_theorem)) and 2) doesn't use a full set of regression variables for the auto.arima model, so it isn't necessarily a fair comparison in my opinion.

Bottom line IMO: if you don't know what you're doing and have daily data, prophet is a great choice. If you want to tweak the models and are willing to build useful features and use an ensemble, use all of the tools.

**Comment by bletham** [comment]
@dashaub is definitely right, it would depend a lot on the problem. Prophet will do well on problems with a piecewise trend and multiple seasonality. An arima model can fit a broader range of time series, but likely would require more expert tuning.
